ezyrb/stlhandler.py

```diff
@@ -0,0 +1,138 @@
+"""
+Derived module from filehandler.py to handle Vtk files.
+"""
+import numpy as np
+import vtk
+import os
+
+
+class StlHandler(object):
+	"""
+	Vtk format file handler class.
+	You are NOT supposed to call directly this class constructor (use
+						Filehandler constructor instead)
+
+	:cvar string filename: name of file to handle
+	:cvar _cached_data: private attribute to store the last polydata
+		processed
+	
```

Comment:
Add a snippet of code as example with the following:
:Example:

	>>> import blabla